.gitignore

```
@@ -21,3 +21,4 @@ test/helm/community_values.yaml
 alfresco-updatecli
 .idea
 .vscode
+.vagrant
```

```suggestion
.vagrant
.venv
venv
```

let's added venv to gitignore as it's a common way to deal with python envs